### PR TITLE
EDA function for LotFrontage, LotArea, MasVnrArea, BsmtFinSF1, BsmtFi…

### PR DESCRIPTION
**Pull request by kylestormcloud**
…nSF2

Some of the features, particularly BsmtFinSF2 and surprisingly LotArea, have weak correlation with SalePrice. For now, I have not deleted any features. This function takes in the complete housing dataframe as a parameter and return a dataframe that contains only my features.

### CODE REVIEW

**Review by srwight — CHANGES_REQUESTED** [review]
Can you pull the visualizations, and just do the feature engineering? Instead, hold on to your visualizations, and use them when we talk / build a presentation.

File: clou.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    import matplotlib.pyplot as plt
+    import seaborn as sns
+    from scipy.stats import skew
+    
+    # Create my dataframe.
+    myDF = pd.DataFrame() # create an empty dataframe to store my features
+    myFeatures = ['SalePrice','BsmtFinSF2', 'BsmtFinSF1', 'MasVnrArea', 'LotArea', 'LotFrontage']
+    for feature in myFeatures:
+        myDF.insert(loc=0,column=feature,value=dataframe[feature]) # insert my features from the master dataframe into my dataframe
+
```
Comment (srwight) [inline clou.py:17]: for 13-16:
Have you considered:
myFeatures = ['BsmtFinSF2', 'BsmtFinSF1', 'MasVnrArea', 'LotArea', 'LotFrontage']
myDF = dataframe[myfeatures]
?
Might be a smidge faster.

Also, don't sweat the sale price. I think we'll separate that out before we ever send it to the functions.

**Review by Hwailaluta — CHANGES_REQUESTED** [review]
1. Return your features. 2. The changes you made in the data before were left out on your second version. 3. Please put imports outside of the function. 4. Be sure to fillna and don't dropna as we need to keep all the rows of data in for now.

One last thing: since you were absent the day we assigned these features, I also did EDA on your features. It doesn't matter now as I can change my code later.

Thank you :)

**Comment by srwight** [comment]
Kyle, when you were out on 12/24 was when we initially assigned these. Since you were out, Desmond picked up yours. We now have two copies of those variables. We should figure out how to handle that.

**Comment by srwight** [comment]
Superseded and outdated. Closing.

**Pull request closed without merging**